src/components/landing/Form.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import H3 from '@material-tailwind/react/Heading3';
+import Paragraph from '@material-tailwind/react/Paragraph';
+import Input from '@material-tailwind/react/Input';
+import Button from '@material-tailwind/react/Button';
+
+export default function Form() {
+    return (
+        <div className="flex flex-wrap justify-center mt-24">
+            <div className="w-full lg:w-8/12 px-4">
+                <div className="relative flex flex-col min-w-0 break-words w-full mb-6">
+                    <div className="flex-auto p-5 lg:p-10">
+                        <div className="w-full text-center">
+                            <H3 color="gray">Want to work with us?</H3>
+                            <Paragraph color="blueGray">
+                                Complete this form and we will get back to you
+                                in 24 hours.
+                            </Paragraph>
+                        </div>
+                        <form onSubmit={(e) => e.preventDefault()}>
+                            <div className="flex gap-8 mt-16 mb-10">
+                                <Input
+                                    type="text"
+                                    placeholder="Full Name"
+                                    color="lightBlue"
+                                />
+                                <Input
+                                    type="email"
+                                    placeholder="Email Address"
+                                    color="lightBlue"
+                                />
+                            </div>
+
+                            <div
+                                className="relative w-full"
+                                style={{ height: '150px' }}
+                            >
+                                <textarea
+                                    placeholder=" "
+                                    className="w-full h-full leading-normal shadow-none outline-none focus:outline-none focus:ring-0 px-0 md-input bg-transparent border-none md-input-light-blue-500"
+                                    style={{ resize: 'none' }}
```

Comment:
You can use one of the classes from here: https://tailwindcss.com/docs/resize instead of the inline style